yt/frontends/sdf/data_structures.py

```diff
@@ -116,6 +116,8 @@ def _parse_parameter_file(self):
         self.domain_dimensions = np.ones(3, "int32")
         if "do_periodic" in self.parameters and self.parameters["do_periodic"]:
```

Comment:
Could we make idioms consistent here while we're at it ?
```suggestion
        if self.parameters.get("do_periodic", False):
```